lib/adapter/freshheads/CssAdapter.ts

```diff
@@ -83,29 +89,31 @@ export default class CssAdapter implements Adapter {
 
         next();
 
-        if (!this.checkMiniCssExtractPluginIsInWebpackConfig(webpackConfig)) {
-            const internalBuilder = new Builder(builderConfig, webpackConfig);
-            internalBuilder.add(new ExtractCssPluginAdapter());
+        if (this.config.sass.enabled) {
```

Comment:
Heb je deze bewust na de `next()` aanroep staan?

Reply:
Nee ik dacht dat dit nodig was omdat de miniCSS eerst ook al na de next stond. Het is toch zo dat dit weer een volgende adapter start dus het na de next moet of werkt het anders ?

Reply:
Alle adapters die na deze adapter zijn toegevoegd, worden uitgevoerd als je de `next()` aanroept. Je kunt daarna code zetten die checkt of eventuele andere adapters dingen aan de webpack config hebben aangepast die je nodig hebt of die niet handig zijn oid.